### ISSUE TITLE
Diff in info message in rcwa-4.6.0/tst/monoids.tst

### ISSUE THREAD

**Issue by olexandr-konovalov** [Stefan-Kohl/stefan-kohl.github.io#5]
As can be seen in the test of the master branch with the most recent set of package updates,
(e.g. https://travis-ci.org/gap-system/gap-docker-pkg-tests-master/jobs/318308599 ), the following happens in all three test scenarios:
```
########> Diff in /home/gap/inst/gap-master/pkg/rcwa-4.6.0/tst/monoids.tst:54
# Input is:
f_zero in M;
# Expected output:
#I  \in for an rcwa mapping <f> and an rcwa monoid <M>
#I  Mult(<f>) = 0, but Mult(<M>) <> 0.
false
# But found:
#I  \in for an rcwa mapping <f> and an rcwa monoid <M>
#I  <M> is class-wise order-preserving, but <f> is not.
false
########
```

**Comment by Stefan-Kohl** [comment]
On the one hand, this indicates just that another applicable criterion is used to show that f_zero is not an element of M. But on the other, this may indicate a problem, since the criterion "Mult(f) = 0, but Mult(M) <> 0" is checked before the criterion "M is class-wise order-preserving, but f is not", but apparently found to be not applicable in the test with the master branch. Since everything works well in 4.8.8, my suspicion would be that there is a chance that the problem is elsewhere in GAP.  -- Can you manually check what Mult(f_zero) and Mult(M) return in the master branch? Or another question - what gives Lcm(0,1) in the master branch? -- If that is not 0, this would likely be the explanation.

**Comment by fingolfin** [comment]
Stefan is dead right: the rewritten LcmInt is wrongly outputting 1 here, not 0. This needs to be fixed in GAP (both stable-4.9 and master)

**Comment by olexandr-konovalov** [comment]
Fixed in GAP in https://github.com/gap-system/gap/pull/2038
